.github/workflows/pmc-ai-pr-summarizer.yml

```
@@ -0,0 +1,45 @@
+# PMC AI PR Summarizer Workflow
+# This GitHub Actions workflow is triggered on pull requests when they are opened or synchronized.
+# It runs on an Ubuntu-latest environment.
+# The workflow has permissions to perform write operations on PR.
+#
+# Steps:
+# 1. Checks out the repository using the actions/checkout@v4.
+# 2. Sets up Node.js version 20 using actions/setup-node@v4.
+# 3. Installs dependencies using npm ci.
+# 4. Executes the build script from the build/index.js file.
+# 5. Displays the Node.js version using node -v.
+name: PMC AI PR Summarizer
+on:
+  workflow_call:
+    workflow_call:
+      secrets:
+        PMCDEVOPS_SSH_PRIVATE_KEY:
```

This doesn't need to be this specific. That's left to the repos that use this workflow.

```suggestion
        SSH_KEY:
```

---

When this is added to the individual repos, in the `secrets` section, you'll map the secret on the repo to the `SSH_KEY` secret defined in this reusable workflow. For example, see how IW's theme uses the `coding-standards.yml` workflow.

---

Thank you make sense. fixed and resolving.